scripts/ci/batch_unit_mutation_lint_targets.sh

```
@@ -0,0 +1,161 @@
+    for i in $(seq 1 60); do
+        if docker exec "$container_name" sh -c "curl -f http://localhost:3000 >/dev/null 2>&1"; then
+            echo "✅ Dev server is responding!"
+            break
+        fi
+        echo "Attempt $i: Dev server not ready yet..."
+        if [ $((i % 10)) -eq 0 ]; then
+            docker exec "$container_name" ps aux 2>/dev/null | grep -E "(next|node)" || echo "No Next.js processes found"
+            docker logs "$container_name" --tail 10
+        fi
+        sleep 3
+        if [ $i -eq 60 ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)